### ISSUE TITLE
Line offset in interpolated symbol layer

### ISSUE THREAD

**Issue by crocovert** [qgis/QGIS#46488]
### Feature description

Hello,

Is it possible to implement line offset in interpolated symbol layer?
I have road layer which are duplicated in both directions, and I would be able to show travel times along the lines which are not identical depending on directions with the possibility to put an offset as it exists in standard line layers

Thanks

**Comment by gioman** [comment]
@crocovert is this a question or a feature request?

**Comment by crocovert** [comment]
It's a feature request

**Comment by chleidersdorff** [comment]
Maybe I dont understand the question properly, but isn't this a cosmetic thing and as far as I know you can achieve this quite easily in QGIS using categorised symbology like below image (Green is original line, black is offset line):

![image](https://user-images.githubusercontent.com/52182284/146132495-a0eb4015-64f2-499c-8759-ced03e0384ca.png)

**Comment by crocovert** [comment]
Yes of, course, but I want to be able to change color depending of travel times along the line depending (I have travels times at the beginning of the line and  travel times at the end of the lines) , and those travel times are different on both directions and in interpolated symbol layer there is no offset parameter
